include/flamegpu/sim/AgentLoggingConfig.h

```diff
@@ -102,63 +102,6 @@ class AgentLoggingConfig {
     bool &log_count;
 };
 
-
```

Comment:
I moved this block into `AgentLoggingConfig_Reductions.h`, to avoid two headers trying to include each other.